api/src/App/Routes.php

```diff
@@ -6,6 +6,7 @@
 $app->get('/member/{name}/status', 'App\Controller\Member\GetStatus');
 $app->post('/member[/]', 'App\Controller\Member\PostMember');
 $app->post('/member/{name}/password', 'App\Controller\Member\PostPassword');
+$app->put('/member/{name}/password/recovery', 'App\Controller\Member\PutPassword');
```

Comment:
All of these should also change eventually to be by ID. But that can be done separately.